StompClientLib/Classes/StompClientLib.swift

```diff
@@ -70,7 +70,7 @@ public protocol StompClientLibDelegate {
 public class StompClientLib: NSObject, SRWebSocketDelegate {
     var socket: SRWebSocket?
     var sessionId: String?
-    var delegate: StompClientLibDelegate?
+    weak var delegate: StompClientLibDelegate?
```

Comment:
@tyh94 

if you want to add `weak` reference, you should make `StompClientLibDelegate` inherit `class`

> 'weak' must not be applied to non-class-bound 'AbuseReporterUIDelegate'; consider adding a protocol conformance that has a class bound

```swift
public protocol StompClientLibDelegate: class { ... }
```